src/scripts/main.js

```diff
@@ -1,3 +1,41 @@
 'use strict';
 
-// write code here
+const nodes = document.querySelectorAll('li');
+const list = document.querySelector('ul');
+
+const frag = document.createDocumentFragment();
+
+const sortList = function(someList) {
```

Comment:
Your function is doing too much, it is not only sorting data but also manipulates the DOM. you can split it into two functions. More over, one of them you will be able to use in getEmployees function.